docs/transaction-api.md

```diff
@@ -4,19 +4,19 @@
 
 ### Payment creation
 
-A user creates a payment by passing a `PaymentIntent` to `Mutation.createOutgoingPayment`. The payment is created in the `PENDING` state.
+A user creates a payment by passing a `PaymentIntent` to `Mutation.createOutgoingPayment`. The payment is created in the `pending` state.
```

Comment:
@sabinebertram re: payment state upper/lower case:

The OP spec uses lower-case for the JSON-string states. But this document is describing the GraphQL API, and GraphQL's convention for enums is all-caps, so I lean towards that.

Reply:
I forgot that this has come up before:
https://github.com/interledger/rafiki/pull/214#discussion_r787000047

I'm leaving states as all-caps for now.
For #252, we can convert case between the `OutgoingPayment` and either the Open Payments API endpoints (upper -> lower) or graphql (lower -> upper).